pkg/k8shandler/rbac.go

```diff
@@ -0,0 +1,106 @@
+package k8shandler
+
+import (
+	"fmt"
+
+	v1alpha1 "github.com/openshift/elasticsearch-operator/pkg/apis/elasticsearch/v1alpha1"
+	"github.com/openshift/elasticsearch-operator/pkg/utils"
+	"github.com/operator-framework/operator-sdk/pkg/sdk"
+	errors "k8s.io/apimachinery/pkg/api/errors"
+)
+
+func CreateOrUpdateRBAC(dpl *v1alpha1.Elasticsearch) error {
+
+	owner := asOwner(dpl)
+
+	// elasticsearch RBAC
+	elasticsearchRole := utils.NewClusterRole(
+		"elasticsearch-metrics",
+		utils.NewPolicyRules(
+			utils.NewPolicyRule(
+				[]string{},
+				[]string{},
+				[]string{},
+				[]string{"get"},
+				[]string{"/metrics"},
+			),
+		),
+	)
+
+	addOwnerRefToObject(elasticsearchRole, owner)
+
+	if err := sdk.Create(elasticsearchRole); err != nil && !errors.IsAlreadyExists(err) {
```

Comment:
can you update this so that we have a second `if` to check if this already exists for readability?